### ISSUE TITLE
Squished full-width character rendering

### ISSUE THREAD

**Issue by Fingercomp** [MightyPirates/OpenComputers#3311]
The current (OC 1.7.5, MC 1.12.2) font renderer draws full-width characters weirdly. Here's an example:

![OpenComputers's renderer](https://i.imgur.com/uVPcmzW.png)

The issue becomes prominent if you compare this screenshot to another renderer:

![An emulator's font renderer](https://i.imgur.com/Fa8PNI7.png)

As you can see, the characters are indeed thinner — they are squished horizontally (by 2 pixels, I believe).

The second screenshot was made using an OC emulator that implements its own font renderer. It has the same `font.hex` file as OpenComputers.

**Comment by payonel** [comment]
what is the 2nd one? ocemu or ocvm or what?

**Comment by Fingercomp** [comment]
It's neither — I used a different (very WIP at the moment) emulator. But I've just tried pasting the "good morning" string into an OCEmu window, and this is what I've got:

![OCEmu](https://i.imgur.com/VfHzLzT.png)

In other words, it draws characters correctly, as shown on the second screenshot in the original post.

**Comment by payonel** [comment]
looks like something your game client is doing, not something we do
optifine?

here is a screen shot from the game:
![image](https://user-images.githubusercontent.com/6590223/83960255-7eac9d80-a83b-11ea-9710-cd0749762d3b.png)

**Comment by Fingercomp** [comment]
No, I don't have OptiFine. I've also asked 3 other people (Windows, Linux; AMD, Nvidia, Intel) to check if they had this bug, and they did (and I've made sure they also don't have OptiFine).

We've found out that the bug vanishes if you set the GUI Scale option to "Large". Which makes sense since the screen GUI gets upscaled. However, at the "normal" GUI scale, the screen GUI size is exactly the same as the emulator's, and the emulators prove it is possible to draw the fonts correctly. So I insist it is an OC bug.
